### DISCUSSION TITLE
Habit tracker

### DISCUSSION THREAD

**Discussion by alephtaph** [BryanJenksCommunity/FAQ#116] [Q&A] [answered]
Hi Bryan
In your template pack you have your "Daily" template with a link to your habit tracker. Would you mind sharing how you have set up a habit tracker in Obsidian? I'm new and trying to figure out all bells and whistles.
Thank you.

**Comment by tallguyjenks** [comment, marked as answer]
I dont use it anymore, but it was just markdown tables, not very robust at all. These days I track some stuff in the YAML front matter and use the obsidian tracker plugin but that's about it honestly
